source/app.css

```diff
@@ -0,0 +1,53 @@
+cloudflare-app[app-id="a-better-browser"] {
```

Comment:
We can preserve most of the compatibility by adding the legacy element IDs.
They won't have any effect on the current styles and will let developers keep their previous CSS selectors.